pkg/cloud-provider/cloudapi/azure/azure_compute.go

```diff
@@ -244,22 +246,25 @@ func (computeCfg *computeServiceConfig) DoResourceInventory() error {
 	return nil
 }
 
-func (computeCfg *computeServiceConfig) SetResourceFilters(selector *v1alpha1.CloudEntitySelector) {
+func (computeCfg *computeServiceConfig) AddSelectors(selector *v1alpha1.CloudEntitySelector) error {
 	subscriptionIDs := []string{computeCfg.credentials.SubscriptionID}
 	tenantIDs := []string{computeCfg.credentials.TenantID}
 	locations := []string{computeCfg.credentials.region}
+
 	if filters, found := convertSelectorToComputeQuery(selector, subscriptionIDs, tenantIDs, locations); found {
```

Comment:
Add a TODO: update convertSelectorToComputeQuery to return an error, which will be updated in the CES status field.

Reply:
There is a TODO added already.